floats/parse_example_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this code is governed by a BSD-style
+// license that can be found in the LICENSE file
+
+package floats_test
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"strings"
+
+	"gonum.org/v1/gonum/floats"
+	"gonum.org/v1/gonum/stat"
+)
+
+func ExampleParseWithNA() {
+	const data = `6
+NA
+4
+`
+
```

Comment:
It would be nice if this had some comment like 
"Compute the mean of a list of numbers, ignoring missing entries"